### PR TITLE
Added minimal make recipe

### PR DESCRIPTION
**Pull request by hoodmane**
Added rule so that `make minimal` does `PYODIDE_PACKAGES="micropip" make`.

### CODE REVIEW

**Comment by dalcde** [comment]
See also #658

**Comment by hoodmane** [comment]
Alternatively, maybe this rule could be called "nopackages".
Relatedly: Is there a way to get docker to remember by repl history between sessions?

File: Makefile [hunk 1]
```diff
@@ -305,3 +305,6 @@ FORCE:
 
 check:
 	./tools/dependency-check.sh
+
+minimal :
+	PYODIDE_PACKAGES="micropip" make
```
Comment (rth) [inline Makefile:310]: LGTM, but there is something in your editor configuration that removes newlines at the end of files.  https://unix.stackexchange.com/a/18789

You can [use pre-commit](https://pyodide.readthedocs.io/en/latest/contributing.html#development-workflow) which [would fix it](https://github.com/iodide-project/pyodide/blob/41592637e6a5231d1fa74235ec5f46919c18e25d/.pre-commit-config.yaml#L7).

Comment (hoodmane) [reply Makefile:310]: I installed pre-commit, but it always failed so I uninstalled it. Probably should open an issue.

Comment (hoodmane) [reply Makefile:310]: Wow I love shit like this: "A file that is not empty and does not end with a newline is therefore not a text file." Very interesting.

**Comment by hoodmane** [comment]
@dalcde Could you merge this? It would save me a fair amount of typing and @rth already signed off on it.

**Comment by dalcde** [comment]
Can you add back the final newline?

**Comment by hoodmane** [comment]
I guess this is another issue with `make lint` that it does not reject missing final end of line.

**Pull request merged**